qualtran/bloqs/sorting.py

```diff
@@ -92,7 +108,7 @@ class BitonicSort(Bloq):
 
     @property
     def signature(self):
-        return Signature([Register("input", bitsize=self.bitsize, shape=(self.bitsize,))])
+        return Signature([Register("input", bitsize=self.bitsize, shape=(self.k,))])
```

Comment:
@fdmalone can you confirm that this is the correct signature

Reply:
yikes